tests/testthat/test-driver-rds.R

```diff
@@ -228,3 +228,16 @@ test_that("vectorised exists", {
   st$set("xx", 2)
   expect_equal(st$exists(c("x", "xx")), c(TRUE, TRUE))
 })
+
+test_that("change directories and access same storr", {
+  x <- storr_rds("my_storr")
+  x$set("a", 1)
+  expect_equal(x$list(), "a")
+  subdir <- "subdir"
+  dir.create(subdir)
+  setwd(subdir)
```

Comment:
Can you change this to use

```r
owd <- setwd(subdir)
on.exit(setwd(owd))
```

which will ensure that test failure leaves things in a sensible state